File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/TypeTemplateService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbTypeTemplate;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface TypeTemplateService {
    PageResult search(TbTypeTemplate tbTypeTemplate, int page, int rows);

    void dele(Long[] ids);

    void save(TbTypeTemplate tbTypeTemplate);

    TbTypeTemplate findOne(Long id);

    List<Map> getTypeList();
}

File: pinyougou-manager-web/src/main/webapp/js/service/typeTemplateService.js
app.service('typeTemplateService',function ($http) {
    this.search = function (searchEntity,page,rows) {
        return $http.post('../typeTemplate/search.do?rows='+rows+"&page="+page,searchEntity);
    }
    this.dele = function (selectids) {
        return $http.post('../typeTemplate/dele.do',selectids);
    }
    this.save=function (entity) {
        return $http.post('../typeTemplate/save.do',entity);
    }
    this.findOne=function (id) {
        return $http.get('../typeTemplate/findOne.do?id='+id);
    }
    this.getTypeList = function () {
        return $http.get('../typeTemplate/getTypeList.do');
    }
})